demos/SDO-OH-EXF-dev-simple/README.md

```diff
@@ -0,0 +1,103 @@
+``` shell
+apt-get -y update
+apt-get -y upgrade
+apt-get -y install jq
+```
+
+### Step Two: Install Open Horizon
+
+> If Open Horizon is already installed and configured properly, skip step two and go straight to step three instead.
+
+* **Optional** - Determine the VM's local IP address and `export HZN_LISTEN_IP=<local-IP-address-here>` if the network connection is bridged.  If the IP address is accessible outside of the VM, then you will be able to connect to any exposed ports from outside the VM.  Without this step, the services will only be available inside the VM.
+* Install the all-in-one Open Horizon instance by following [these instructions](https://github.com/open-horizon/devops/tree/master/mgmt-hub).
+* **NOTE** Save the credentials shown at the end of the installation process.  You will need these later.
+* Set the following environment variables to connect to the Exchange and to allow publishing.  Replace the `<values>` with the actual value indicated.  Also add them to your shell's RC file if you'd like them to persist beyond this session.
+``` shell
+export HZN_ORG_ID=myorg
+export HZN_EXCHANGE_USER_AUTH=admin:<insert value from installation credentials>
+export HZN_DEVICE_TOKEN=<insert value from installation credentials or use random string>
+export HZN_DEVICE_ID=node1
+export HZN_EXCHANGE_URL=http://<local-IP-address-here>:3090/v1/
+export HZN_FSS_CSSURL=http://<local-IP-address-here>:9443/
+```
+
+### Step Three: Configure and publish the definition files for the EXF Service
+
+> NOTE: this is based on the [Open Horizon / EdgeX Foundry integration project](https://github.com/edgexfoundry-holding/open-horizon-integration/blob/master/hub/README.md).
+
+* Clone the [ORRA repository](https://github.com/edgexfoundry-holding/orra) and navigate to the SDO-OH-EXF demo folder:
+``` shell
+git clone https://github.com/edgexfoundry-holding/orra.git
+cd orra/demos/SDO-OH-EXF-dev-simple
+```
+* Publish the service and pattern:
+``` shell
+hzn exchange service publish -P -f configuration/service.json
+hzn exchange pattern publish -f configuration/pattern.json
+```
+* Create the temporary and configuration folders, copy the EdgeX Foundry service configuration patterns to their expected location:
+``` shell
+mkdir -p /var/run/edgex/logs
+mkdir -p /var/run/edgex/data
+mkdir -p /var/run/edgex/consul/data
+mkdir -p /var/run/edgex/consul/config
+mkdir -p /root/res
+chmod -R a+rwx /var/run/edgex
+chmod -R a+rwx /root/res
+cp res/* /root/res
+```
+* Publish the deployment policy:
+``` shell
+hzn exchange deployment addpolicy -f configuration/deployment.policy.json myorg/policy-com.github.joewxboy.horizon.edgex_1.0.1
+```
+
+### Step Four: Edit and run the SDO script
+
+* Go back to your starting folder:
+``` shell
+cd ../../..
+```
+* Edit the `test-sdo.sh` script to change the following lines:
+    * Approximately at line 203: `hzn voucher import /var/sdo/voucher.json --policy node.policy.json`
```

Comment:
Specifying a line number is bound to quickly be wrong, and as a matter of fact it already is. :-) I suggest instead:
```suggestion
    * Towards the end, the line: `hzn voucher import /var/sdo/voucher.json --policy node.policy.json`
```